packages/realm-web/src/node/index.ts

```diff
@@ -0,0 +1,61 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2020 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2020 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+import { setEnvironment, Environment } from "../environment";
+import { MemoryStorage } from "../storage";
+
+const environment: Environment = {
+    console,
+    defaultStorage: new MemoryStorage(),
+    openWindow: url => {
+        console.log(`Please open this URL: ${url}`);
```

Comment:
Should we use something like https://github.com/sindresorhus/open? Or maybe do something with a headless browser?

If not, should this just throw since it isn't able to fulfill the contract of opening a window?

Reply:
I would only really expect this to be called from integration tests when they're running from node.
In that case I don't know what the expected behaviour would be - I can see how it would be annoying to get UI spawned when running on CI for example.
I believe it does fulfill the contract. `openWindow` is allowed to return `null` if no windows handle was produced.